pantheon/src/main/java/tech/pegasys/pantheon/controller/MainnetPantheonController.java

```diff
@@ -59,6 +59,7 @@
 
   private static final Logger LOG = LogManager.getLogger();
   public static final int MAINNET_NETWORK_ID = 1;
+  public static final int ROPSTEN_NETWORK_ID = 3;
```

Comment:
Is `MainnetPantheonController` a reasonable place to define this?

Reply:
Doesn't really fit all that well. I suspect it would be better if all three of MainNet, Rinkeby and Ropsten network IDs were moved to the EthNetworkConfig class. It already has the statics for genesis configs and is the only place those constants are used.